src/java/fr/paris/lutece/portal/web/user/AdminUserJspBean.java

```diff
@@ -1698,7 +1698,7 @@ public String doModifyAdminUserRights( HttpServletRequest request ) throws Acces
             }
         }
 
-        if ( ( user != null ) && ( userCurrent != null ) && ( user.getUserId( ) == userCurrent.getUserId( ) ) )
+        if ( user.getUserId( ) == userCurrent.getUserId( ) )
```

Comment:
I guess `user` and `userCurrent` cannot be null here because if not `isUserAuthorizedToModifyUser( userCurrent, user )` would have failed ? Maybe add asserts to make that clear ?

Reply:
Modified isUserAuthorizedToModifyUser to return false if either use is null